smithy-typescript-codegen/src/test/resources/software/amazon/smithy/typescript/codegen/endpointsV2/endpoints.smithy

```diff
@@ -6,10 +6,7 @@ namespace smithy.example
   "version": "1.3"
   "parameters": {
     "Region": {
-      "builtIn": "AWS::Region",
       "type": "String",
-      "required": true,
-      "default": "us-east-1",
```

Comment:
Does `required` and `default` need to be removed?

Reply:
It configures the parameters to be relevant for the rules.